fk/fk-parent/src/main/java/se/inera/intyg/common/fkparent/model/converter/RespConstants.java

```diff
@@ -47,23 +47,31 @@ public final class RespConstants {
     public static final String GRUNDFORMEDICINSKTUNDERLAG_TELEFONKONTAKT_PATIENT_SVAR_JSON_ID_1 = "telefonkontaktMedPatienten";
     public static final String GRUNDFORMEDICINSKTUNDERLAG_BESKRIVNING_DELSVAR_JSON_ID_1 = "annatGrundForMUBeskrivning";
     public static final String MOTIVERING_TILL_INTE_BASERAT_PA_UNDERLAG_ID_1 = "motiveringTillInteBaseratPaUndersokning";
+    public static final String MOTIVERING_TILL_INTE_BASERAT_PA_UNDERLAG_DELSVAR_ID_1 = "1.4";
+    public static final String MOTIVERING_TILL_INTE_BASERAT_PA_UNDERLAG_TEXT = "DFR_1.4.RBK";
```

Comment:
Introduce the texts as constants instead of TEXT-IDs. See comment on PR for Refdata.